src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -213,26 +213,29 @@ namespace
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_7,         L"Coverage" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_8,         L"Depth" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_9,         L"Facing Ratio" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_10,        L"Geometric Normals" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_11,        L"Materials" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_12,        L"Object Instances" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_13,        L"Original Shading Normals" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_14,        L"Primitives" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_15,        L"Ray Spread" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_16,        L"Screen Space Wireframe" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_17,        L"Shading Normals" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_18,        L"Sides" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_19,        L"Tangents" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_20,        L"UV Coordinates" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_21,        L"World Space Position" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_22,        L"World Space Velocity" },
-        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_23,        L"World Space Wireframe" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_4,         L"Assemblies" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_5,         L"Assembly Instances" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_6,         L"Barycentric Coordinates" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_7,         L"Bitangents" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_8,         L"Coverage" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_9,         L"Depth" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_10,        L"Facing Ratio" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_11,        L"Geometric Normals" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_12,        L"Materials" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_13,        L"Object Instances" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_14,        L"Objects" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_15,        L"Original Shading Normals" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_16,        L"Primitives" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_17,        L"Ray Spread" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_18,        L"Screen-Space Velocity" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_19,        L"Screen-Space Wireframe" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_20,        L"Shading Normals" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_21,        L"Sides" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_22,        L"Tangents" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_23,        L"UV Coordinates" },
+        { IDS_RENDERERPARAMS_SHADER_OVERRIDE_24,        L"World Space Position" },
```

Comment:
For consistency's sake, "World-Space" (below as well).

Reply:
indeed